src/ngSanitize/filter/linky.js

```diff
@@ -137,8 +163,19 @@ angular.module('ngSanitize').filter('linky', ['$sanitize', function($sanitize) {
     }
 
     function addLink(url, text) {
+      var key;
+      var computedAttributes = {};
       html.push('<a ');
-      if (angular.isDefined(target)) {
+      if (angular.isFunction(attributes)) {
+        attributes = attributes(url);
+      }
+      if (angular.isObject(attributes)) {
+        computedAttributes = attributes;
+        for (key in computedAttributes) {
+          html.push(key + '="' + computedAttributes[key] + '" ');
```

Comment:
My understanding was that this is dealt with by $sanitize, but anyone correct me if I'm wrong.